docs/how-to/test-update-system-locally.md

```diff
@@ -0,0 +1,39 @@
+# Testing update system locally
+
+This instruction covers how to test software update locally:
+
+  1. Prepare and propose software update
+      a. Prepare alt configuration with changed application version constant
+      b. Propose updated with script that simply replaces one configuration file (in setup) with other
+  2. Launch wallet node with launcher, Daedalus (or Daedalus emulator)
+      a. When update arrives Deadalus to finish, launcher to restart Daedalus + node
```

Comment:
Sounds like broken English.

Reply:
It does, I did some changes but don't really know what to write